application/helpers/questionHelper.php

```diff
@@ -1644,28 +1644,10 @@ public static function getQuestionThemeAttributeValues($type, $sQuestionThemeNam
             $aQuestionAttributes[$aCoreAttribute['name']] = $aCoreAttribute;
         }
 
-        /**
-         * New event to allow plugin to add own question attribute (settings)
```

Comment:
Since old plugin using this event
1. Can broke with debug
2. Happen only in some condition (didn't work … with all question type)
3. value are not saved

I choose to remove